docs/firstobject.md

```diff
@@ -16,40 +17,39 @@ Note: This guide will assume Archipelago is running on `http://localhost:8001`,
 
 Start by opening `http://locahost:8001` in your favourite Web Browser.
 
-![Welcome](../imgs/Welcome.jpg)
+![Welcome](../imgs/ingest-first-object_beta2/01_log-in.jpg)
 
 Your Demo deployment will have a fancy Home page with some banners and a small explanation of what Archipelago is and can do. Feel free to read through that now or later.
 
-Click on `Log in` and use your `demo` credentials from them deployment guide.
+Click on `Log in` in the top left corner and use your `demo` credentials from the deployment guide.
```

Comment:
Good. So, question for beta3. Since we are providing screenshots, should we keep telling people where in the page something is to be found at that level of specificity? Some edge cases. If people are seeing the docs in a small screen v/s a mobile device. Will this stay true? Just a thought to be discussed for documentation on beta3. Also because theme defines a lot.

Reply:
Yes, I agree it's something to consider for the future iterations. For this particular update, I was trying to slim down some of the language and the directions more concise in order to mitigate the amount of text edits and screenshot updates (which are just inevitable i suppose) in the future. But I agree, explicitly giving direction on where to navigate the site is subject to change with time.